CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 
+<!-- 8.1.0-beta3 START -->
+
+# 8.1.0-beta3 (2021-07-29)
+
+### Features and enhancements
+
+* **Alerting:** Support label matcher syntax in alert rule list filter. [#36408](https://github.com/grafana/grafana/pull/36408), [@nathanrodman](https://github.com/nathanrodman)
+* **IconButton:** Put tooltip text as aria-label. [#36760](https://github.com/grafana/grafana/pull/36760), [@tskarhed](https://github.com/tskarhed)
+* **Live:** Experimental HA with Redis. [#36787](https://github.com/grafana/grafana/pull/36787), [@FZambia](https://github.com/FZambia)
+* **UI:** FileDropzone component. [#36646](https://github.com/grafana/grafana/pull/36646), [@zoltanbedi](https://github.com/zoltanbedi)
```

Comment:
What does this mean without opening up the PR and reading it verbatim?

Reply:
Looks like this could read, "**UI:** Add `react-dropzone` package with FileDropzone component.

Reply:
@zoltanbedi, what do you think? If you agree, please change upstream.

Reply:
hmm. I wouldn't emphasise that it uses react-dropzone under the hood. I'm not sure how I would describe that we added a new component that is capable of file uploading.

Reply:
"Add capability to upload files"?

Reply:
**UI:** Add FileDropzone component that is capable to upload files. ?